### PR TITLE
LTS Release/5.11.4

### PR DESCRIPTION
**Pull request by kopijunkie**
### Changed

- Public: Reduce resolution to VGA if browser does not support recording videos in WebM format (e.g. Safari 14.x that only supports MP4 format) to avoid large video files being created.
- Internal: Update SDK's Publish Release workflow to not use the now deprecated `set-env` command.
- Internal: Fix cross-device SMS number input bundle import that broke when using newer versions of `@babel/preset-env`.

### CODE REVIEW

**Comment by kopijunkie** [comment]
🚀  PR Surge link: https://1504-pr-onfido-sdk-ui-onfido.surge.sh/

**Review by ethan605 — APPROVED** [review]
LGTM!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "5.11.3",
+  "version": "5.11.4-rc.4",
```
Comment (eraxyz) [inline package.json:3]: When does this become 5.11.4?

**Review by eraxyz — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (kopijunkie) [reply package.json:3]: When we make the actual release, the version will be updated to `"5.11.4"` here.

**Review by Phoebe-B — APPROVED** [review]

**Comment by kopijunkie** [comment]
Closing as part of LTS patch fix post release process.

**Pull request closed without merging**